.ci/wercker.sh

```diff
@@ -81,7 +81,8 @@ no-error-hibernate-search)
   CS_POM_VERSION=$(mvn -e -q -Dexec.executable='echo' -Dexec.args='${project.version}' \
                      --non-recursive org.codehaus.mojo:exec-maven-plugin:1.3.1:exec)
   echo CS_version: ${CS_POM_VERSION}
-  checkout_from https://github.com/hibernate/hibernate-search.git
+  # till https://github.com/hibernate/hibernate-search/pull/1698
+  checkout_from https://github.com/ps-sp/hibernate-search.git
```

Comment:
wercker isn't failing and this PR was merged. We don't need these CI modifications anymore.

Reply:
@rnveach Actually wercker is failing.

Also, there's a little issue. There were actually two counts of the same code formatting problem with hibernate. The other one was realized in https://github.com/checkstyle/checkstyle/pull/4247#issuecomment-400529066. My corresponding PR for hibernate was merged before I could have updated it with the second correction. I will have to make one more PR there. Also, wercker was failing when I'd pushed previously, may be since I'd not rebased over the latest master.

Reply:
Yes, was failing because it was an old version and someone was using the new module we released.